private/pkg/netrc/netrc_unix_test.go

```diff
@@ -140,6 +143,54 @@ func TestPutMachines(t *testing.T) {
 	)
 }
 
+// https://github.com/bufbuild/buf/issues/611
+func TestPutLotsOfBigMachinesSingleLineFiles(t *testing.T) {
+	t.Parallel()
+	// TODO: at size 10, this test passes, at size 100, it fails
+	size := 100
+	password := strings.Repeat("abcdefghijklmnopqrstuvwxyz", size)
```

Comment:
Note if you are debugging, you might want to decouple the number of machines from the size of the password to try to better isolate the problem.